### DISCUSSION TITLE
v5.60.0

### DISCUSSION THREAD

**Discussion by sokra** [webpack/webpack#14563] [General]
# Features

* Allow to pass more options to `experiments.lazyCompilation`. e. g. port, https stuff

# Bugfixes

* fix `output.hashFunction` used to persistent caching too
* Initialize `buildDependencies` Set correctly when loaders are added in `beforeLoaders` hook

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v5.60.0'>v5.60.0</a>.</em>

**Comment by GetBentGaming** [comment]


**Reply by GetBentGaming** [reply]
Not 11,986

**Reply by GetBentGaming** [reply]
[@GetBentGaming on Google Bing You name it I have a site and none of them are working ever since GitHub]()

**Reply by GetBentGaming** [reply]
Seriously this is out off hand I work just as hard as y'all just doing other stuff
